README.md

```diff
@@ -31,22 +31,22 @@ using the System Properties menu (on Windows).
 
 ### Install dependencies
 
-Set up a virtual environment using [Anaconda](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#creating-an-environment-with-commands) or [virtualenv](https://docs.python.org/3/library/venv.html).
-
-#### Anaconda
+Create and activate a conda environment using [Anaconda](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#creating-an-environment-with-commands):
```

Comment:
I see that before we had instructions for virtualenv wont we support that? Still a lot of users use virtualenv

Reply:
I'm also torn between the prescriptivism of recommending just conda or also keeping in instructions for virtualenv, since we do have that mentioned in our [docs](https://docs.snowflake.com/en/developer-guide/snowpark/python/setup#prerequisites).

Reply:
For the VS Code integration we went with just conda for simplicity of ensuring environment similarity.

Reply:
Also torn, my concern was that if someone wants to use a package from PyPi when they deploy to the Snowflake server, the package will come from Conda. So I figured it'll be good to get them on the Conda train sooner so their packages are consistent